docs/source/2019Barry.rst

```diff
@@ -106,13 +109,16 @@ Hurricane Track and Intensity
 
       Hurricane tracks from GFS.v16.0.10 (red line) and Best Track (black line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
 
-    * GFS.v16.0.10 generates right-of-track bias. 
+    * GFS.v16.0.10 generated right-of-track bias. 
```

Comment:
Same here and in many other places. As long as this version doesn't change (it won't), it will continue to produce a right-of-track bias.